### PR TITLE
code review for sdk 6.5.x

### PR DESCRIPTION
**Pull request by HuFangshuai**
this PR has been created for banner function support in our AdMob adapter.

### CODE REVIEW

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 1]
```diff
@@ -212,7 +209,8 @@ public void requestBannerAd(Context context,
 
         mPlacementForPlay = mVungleManager.findPlacement(mediationExtras, serverParameters);
         boolean isPlacementValid = !TextUtils.isEmpty(mPlacementForPlay);
-        boolean isBannerSizeValid = hasBannerSizeAd(adSize);
+        mAdSize = getVungleAdSize(adSize);
+        boolean isBannerSizeValid = mAdSize != AdConfig.AdSize.VUNGLE_DEFAULT;
```
Comment (jiancchen) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: @HuFangshuai take a look at using AdConfig.AdSize.isBannerSize

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 2]
```diff
@@ -39,10 +36,10 @@
 
     //banner
     private volatile RelativeLayout adLayout;
-    private VungleNativeAd vungleNativeAd;
```
Comment (jiancchen) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: @HuFangshuai need to keep VungleNativeAd for MREC Implementation

Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: fixed

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java (hunk 1 above)
Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: fixed

File: ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java [hunk 3]
```diff
@@ -343,13 +375,19 @@ public View getBannerView() {
...
     }
 
-    private boolean hasBannerSizeAd(AdSize adSize) {
-        if (300 == adSize.getWidth() && 250 == adSize.getHeight()) {
-            adConfig.setAdSize(VUNGLE_MREC);
-            return true;
+    private AdConfig.AdSize getVungleAdSize(AdSize adSize) {
+        AdConfig.AdSize adSizeType = null;
+
+        if (VUNGLE_MREC.getWidth() == adSize.getWidth() && VUNGLE_MREC.getHeight() == adSize.getHeight()) {
```
Comment (jiancchen) [inline ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java]: @HuFangshuai I can't find any dimensions in com.google.android.gms.ad.AdSize for Banner short btw. Is this why we can't do 

```
        if (adSize == AdSize.BANNER) {
            adSizeType = BANNER;
        } else if (adSize == AdSize.MEDIUM_RECTANGLE) {
```

File: ThirdPartyAdapters/vungle/vungle/build.gradle [hunk 4]
```diff
@@ -37,8 +37,12 @@ android {
 }
 
 dependencies {
-    implementation ('com.vungle:publisher-sdk-android:6.4.11') {
-        transitive=true
+    if (gradle.ext.has('dependOnVungleSdkProject') && gradle.ext.dependOnVungleSdkProject) {
+        implementation project(':vungle-android-sdk:publisher-sdk-android')
+    } else {
+        implementation ('com.vungle:publisher-sdk-android:6.4.11') {
```
Comment (palrahul) [inline ThirdPartyAdapters/vungle/vungle/build.gradle]: point to 6.5.0-RC3 from jitpack

Comment (HuFangshuai) [reply ThirdPartyAdapters/vungle/vungle/build.gradle]: fixed

**Pull request merged**